components/site/header.tsx

```diff
@@ -17,7 +17,7 @@ const Header = (props: HeaderProps) => {
 
   return (
     <Box my="600">
-      <Box as="header" height="650" display="flex" justifyContent="space-between" mb="400">
+      <Box as="header" height="650" display="flex" justifyContent="space-between" mb="500">
```

Comment:
We probably also need to apply a new logo in the footer too. Not this one, but a white one.